react/src/functions.tsx

```diff
@@ -34,34 +37,13 @@ export function toKeyValue(items: string[]) {
 }
 
 /**
- * Return a date string in the format "YYYY-MM-DD", if possible.
- *
- * @param {string} date Datestring of the form: "Day, DD Mon YYYY HH:MM:SS GMT"
+ * Convert an ISO datetime to human-readable format in the user's locale.
  */
 export function formatDateString(date: string) {
-    const months = [
-        "Jan",
-        "Feb",
-        "Mar",
-        "Apr",
-        "May",
-        "Jun",
-        "Jul",
-        "Aug",
-        "Sep",
-        "Oct",
-        "Nov",
-        "Dec",
-    ];
-    // Pretty general datestring because we trust the server to send a good one
-    const regex = /^[A-Z][a-z]{2}, (\d{2}) ([A-Z][a-z]{2}) (\d{4}) \d{2}:\d{2}:\d{2} GMT$/;
-    const result = regex.exec(date);
-    if (result) {
-        let [year, month, day] = [result[3], "" + (months.indexOf(result[2]) + 1), result[1]];
-        if (month.length < 2) month = "0" + month;
-        return [year, month, day].join("-");
-    }
-    return date;
+    dayjs.extend(utc);
+    dayjs.extend(LocalizedFormat);
```

Comment:
If these are plugins, shouldn't they be added globally instead of again and again on each function call?